### DISCUSSION TITLE
Blanks icons after flashing screen.

### DISCUSSION THREAD

**Discussion by taylotr** [Jyers/Marlin#1889] [General Discussion]
Hello all,

I have compiled the latest firmware to support the Micro-Swiss Direct Drive.  While doing it, I thought I would update the icons as well.  I pulled them from the source and renamed the folder to DWIN_SET as required and after restarting the printer the screen doesn't show the icons.  I have attached a screenshot for reference.

![IMG_20220306_152909](https://user-images.githubusercontent.com/3688242/156941044-58239d9e-bb6c-4fc7-b002-da4652e1b80f.jpg)

**Comment by taylotr** [comment]
That was it thank you!

I am good to go with that, but just as an additional tidbit:  The printer firmware now:

1. Loads, showing the Creality screen with the progress bar.
2. Shows the icons as expected.
3. Once the icons are showing, it briefly shows the progress bar again over the icons as if the printer is booting still.

After that, once single instance of the progress bar showing over the icons, everything appears to be fine.


Thank you for your help!

**Comment by taylotr** [comment]
I just removed/reinserted my SD Card to print and it appears that the progress bar is triggered when the printer is reading the card into memory.

**Comment by JCRodzDom** [comment]
I was having the same problem, After many times trying to compiled a the program for a 4.2.7 board, Slice Engineer Thermistor 400 degrees and bump up the temperature of the hot end to 340 degrees,; successfully. 
Now my display is not working properly. Here is how is looks now after the flashing
Can you explain more detailed the steps to fix this problem ? thanks
![Ender 3V2 Display 3_6_2022](https://user-images.githubusercontent.com/39976893/156958256-8063b1f7-f63b-49b5-b9c8-817a41c57ff2.jpg)
.

**Comment by JCRodzDom** [comment]
On Mon, Mar 7, 2022 at 12:33 AM tititopher68-dev ***@***.***>
wrote:

> @taylotr <https://github.com/taylotr>
> If you printer reboot, you have some issue with your files configuration.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Jyers/Marlin/discussions/1889#discussioncomment-2306347>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJQ77PPN4YXKFFHGNBVMKXDU6WPK3ANCNFSM5QBSRW4Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Eng. Juan Carlos Rodríguez Domenech
Cel: 1-501-916-0027

Before you print, please consider the negative environmental impacts
associated with printing.  A reminder from JC Rodz. environmentally
conscious.
CONFIDENTIALITY NOTE: This communication and all attachments included
herein contain information that belongs to JC Rodz. This document is
considered as confidential and legally privileged. If you have received
this communication and your are not recognize that it for you, please
delete it and notify the sender immediately. The distribution of this email
is strictly prohibited without an authorization from the sender. Juan C
Rodz Domenech  has the latest Antivirus software and continuously upgrade
it, but, cannot assure that that this email is virus free such is indeed
the case and disclaim any responsibility.

**Comment by JCRodzDom** [comment]
There is anyway for you to compile for me? I’ve been trying for too long
and really want to finish my project that is actually printing hi temp
material.
Let me know
Thanks

On Mon, Mar 7, 2022 at 7:26 AM Juan Rodriguez ***@***.***>
wrote:

>
>
> On Mon, Mar 7, 2022 at 12:33 AM tititopher68-dev ***@***.***>
> wrote:
>
>> @taylotr <https://github.com/taylotr>
>> If you printer reboot, you have some issue with your files configuration.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Jyers/Marlin/discussions/1889#discussioncomment-2306347>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AJQ77PPN4YXKFFHGNBVMKXDU6WPK3ANCNFSM5QBSRW4Q>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>> You are receiving this because you commented.Message ID:
>> ***@***.***>
>>
> --
> Eng. Juan Carlos Rodríguez Domenech
> Cel: 1-501-916-0027
>
> Before you print, please consider the negative environmental impacts
> associated with printing.  A reminder from JC Rodz. environmentally
> conscious.
> CONFIDENTIALITY NOTE: This communication and all attachments included
> herein contain information that belongs to JC Rodz. This document is
> considered as confidential and legally privileged. If you have received
> this communication and your are not recognize that it for you, please
> delete it and notify the sender immediately. The distribution of this email
> is strictly prohibited without an authorization from the sender. Juan C
> Rodz Domenech  has the latest Antivirus software and continuously upgrade
> it, but, cannot assure that that this email is virus free such is indeed
> the case and disclaim any responsibility.
>
-- 
Eng. Juan Carlos Rodríguez Domenech
Cel: 1-501-916-0027

Before you print, please consider the negative environmental impacts
associated with printing.  A reminder from JC Rodz. environmentally
conscious.
CONFIDENTIALITY NOTE: This communication and all attachments included
herein contain information that belongs to JC Rodz. This document is
considered as confidential and legally privileged. If you have received
this communication and your are not recognize that it for you, please
delete it and notify the sender immediately. The distribution of this email
is strictly prohibited without an authorization from the sender. Juan C
Rodz Domenech  has the latest Antivirus software and continuously upgrade
it, but, cannot assure that that this email is virus free such is indeed
the case and disclaim any responsibility.

**Comment by JCRodzDom** [comment]
OK, I will keep trying compiling, 
So following your link , i will be using the following:
Sources v2.0.2i for STM32F103RCT6 - 256k flash memory - I have to be sure if i got a 256 or a 512k ship
So I will be compiling for the following:
Ender 3 V2 with upgraded motherboard 2.2.7
Motherboard 2.2.7
Slice Engineering thermistor 450 degrees
Manual leveling 3 x 3 minimum or 5 x 5 if possible.
Max temperature 325 degrees

I will keep asking questions,  hope you guys can help. thanks a lot in advance !!!

**Reply by HAPSagan** [reply]
The motherboard should perhaps be 4.2.7? 🤔

**Comment by JCRodzDom** [comment]
Well I try to compile these two releases and they will fail out of the box not even try to make any changes.
Sources :
Sources v2.0.2i for STM32F103RCT6 - 256k flash memory :
[Marlin-bugfix-JyersUI-v202i-RCT6.zip](https://github.com/Jyers/Marlin/files/7937917/Marlin-bugfix-JyersUI-v202i-RCT6.zip)

Sources v2.0.2i for STM32F103RET6 - 512k flash memory :
[Marlin-bugfix-JyersUI-v202i-RET6.zip](https://github.com/Jyers/Marlin/files/7934962/Marlin-bugfix-JyersUI-v202i-RET6.zip)


Any recommendations !!!
